wgsl/index.bs

```diff
@@ -565,37 +567,168 @@ as the first parameter.
+  <tr><td>rg8unorm<td>8unorm<td>r, g<td>vec4&lt;f32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg8snorm<td>8snorm<td>r, g<td>vec4&lt;f32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg8uint<td>8uint<td>r, g<td>vec4&lt;u32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg8sint<td>8sint<td>r, g<td>vec4&lt;i32&gt;(CTF(r), CTF(g), ?, ?)
+</table>
+
+<table class='data'>
+  <caption>32-bit Texel Formats</caption>
+  <thead>
+    <tr><td>Texel format<td>Channel format<td>Channels in memory order<td width="50%">Corresponding shader value
+  </thead>
+  <tr><td>r32uint<td>32uint<td>r<td>vec4&lt;u32&gt;(CTF(r), ?, ?, ?)
+  <tr><td>r32sint<td>32sint<td>r<td>vec4&lt;i32&gt;(CTF(r), ?, ?, ?)
+  <tr><td>r32float<td>32float<td>r<td>vec4&lt;f32&gt;(CTF(r), ?, ?, ?)
+  <tr><td>rg16uint<td>16uint<td>r, g<td>vec4&lt;u32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg16sint<td>16sint<td>r, g<td>vec4&lt;i32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg16float<td>16float<td>r, g<td>vec4&lt;f32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rgba8unorm<td>8unorm<td>r, g, b, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba8snorm<td>8snorm<td>r, g, b, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba8uint<td>8uint<td>r, g, b, a<td>vec4&lt;u32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba8sint<td>8sint<td>r, g, b, a<td>vec4&lt;i32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+</table>
+
+<table class='data'>
+  <caption>64-bit Texel Formats</caption>
+  <thead>
+    <tr><td>Texel format<td>Channel format<td>Channels in memory order<td width="50%">Corresponding shader value
+  </thead>
+  <tr><td>rg32uint<td>32uint<td>r, g<td>vec4&lt;u32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg32sint<td>32sint<td>r, g<td>vec4&lt;i32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rg32float<td>32float<td>r, g<td>vec4&lt;f32&gt;(CTF(r), CTF(g), ?, ?)
+  <tr><td>rgba16uint<td>16uint<td>r, g, b, a<td>vec4&lt;u32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba16sint<td>16sint<td>r, g, b, a<td>vec4&lt;i32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba16float<td>16float<td>r, g, b, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+</table>
+
+<table class='data'>
+  <caption>128-bit Texel Formats</caption>
+  <thead>
+    <tr><td>Texel format<td>Channel format<td>Channels in memory order<td width="50%">Corresponding shader value
+  </thead>
+  <tr><td>rgba32uint<td>32uint<td>r, g, b, a<td>vec4&lt;u32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba32sint<td>32sint<td>r, g, b, a<td>vec4&lt;i32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+  <tr><td>rgba32float<td>32float<td>r, g, b, a<td>vec4&lt;f32&gt;(CTF(r), CTF(g), CTF(b), CTF(a))
+</table>
+
+The following table lists the correspondence between WGSL texel formats and [SPIR-V image formats](https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#_a_id_image_format_a_image_format).
+
+<table class='data'>
+  <caption>Mapping texel formats to SPIR-V</caption>
+  <thead>
+    <tr><td>Texel format<td>SPIR-V Image Format<td>Enabling SPIR-V capability
```

Comment:
it seems to me that we'd need to remove any mention of this SPIR-V capability, because it doesn't translate to what WebGPU can support, see https://gpuweb.github.io/gpuweb/#texture-format-caps

Reply:
This table is for translation _to_ SPIR-V.  Some of the image formats that I think are to be supported by WebGPU  do require the extended image formats capability.

Reply:
I don't think any of our formats require this capability today.

Reply:
Ah, ok.  I've updated to eliminate all those extended formats.